packages/common/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@subql/common",
-  "version": "5.1.5-1",
+  "version": "5.1.5",
```

Comment:
I think this needs to be more than a patch, removing cockroach will not be compatible